docs-content/usecase.html.md.erb

```diff
@@ -17,8 +17,54 @@ etc.
 
 the corresponding a9s service tile will talk to the <b>a9s BOSH for PCF</b> tile and create a BOSH deployment in order to provision virtual machines and to install the required software packages.
 
-## Why we decided not to use the PCF Ops Manager Director for this purpose
+<img src="./images/a9s-bosh-for-pcf-architecture.png" alt="a9s BOSH for PCF Architecture" />
 
-We want to provide the possiblity to choose between the <b>a9s BOSH for PCF</b> and the <b>PCF Op Manager Director</b> in the future. For proof of concepts this would be a useful option, because the <b>a9s BOSH for PCF</b> tile does not need to be installed.
+## Advantages of Using a9s BOSH for PCF
 
-Due to the missing experience with using one BOSH director only for the regular PCF tile deployments as well as for the service instance deployments, we focus our support on using the a9s BOSH for PCF tile for now.
+In general, it is expected to have only a few tiles in your <b>PCF Ops Manager Director</b>,
```

Comment:
Not a huge fan of the "this is expected" formulation. What do you mean here? Are you describing a typical use case?